### PR TITLE
Commentfix ideamanagement jessica

### PR DESCRIPTION
**Pull request by JessicaReistel**
In IdeasManagementView Beschreibung der Methode initializeGrid() geändert

### CODE REVIEW

File: src/main/java/com/example/application/ui/horizontal/community/IdeasManagementView.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
  *  @author Litharshiga Sivarasa, Jessica Reistel, Vanessa Skowronsky
  *  @version 3.0
  *  @since 15.12.2020
- *  @lastUpdated 07.02.2021 by Jessica Reistel
+ *  @lastUpdated 10.02.2021 by Jessica Reistel
```
Comment (anastasiya-jh) [inline src/main/java/com/example/application/ui/horizontal/community/IdeasManagementView.java]: When adding only comments you don't need to change the lastUpdated :') just leave it as it is

**Review by anastasiya-jh — CHANGES_REQUESTED** [review]
The last two method-comments are no Java doc comments, please change that

**Comment by JessicaReistel** [comment]
Hab's geändert :)

**Review by VannaSky — APPROVED** [review]
Dankeschön :-)

**Pull request merged**